src/pykiso/test_coordinator/test_execution.py

```diff
@@ -474,12 +475,19 @@ def execute(
 
         log_file_path = get_logging_options().log_path
         # TestRunner selection: generate or not a junit report. Start the tests and publish the results
+        # pykiso --junit
         if report_type == "junit":
-            junit_report_name = time.strftime(f"%Y-%m-%d_%H-%M-%S-{report_name}.xml")
-            project_folder = Path.cwd()
-            reports_path = project_folder / "reports"
-            junit_report_path = reports_path / junit_report_name
-            reports_path.mkdir(exist_ok=True)
+            report_path = junit_path
+            full_report_path = Path.cwd() / report_path
```

Comment:
That would not work if I drop the location, I will get a absolute path instead

Reply:
what do you mean ? 
I mean I looks weird but if report_path is an abspath Path.cwd()  gets ignored...
tested on windows and linux

Reply:
Really, cool, still please comment on top, because yes, it looks really weird ^^